source/game/field/ObjectCollisionBase.cc

```diff
@@ -177,7 +177,7 @@ void ObjectCollisionBase::getNearestPoint(GJKState &state, u32 idx, EGG::Vector3
     v1 *= 1.0f / sum;
 }
 
-/// @addr{0x808357E4} @addr{0x808359A4}
+/// @addr{0x808357E4}
```

Comment:
Contrary to what the name of this function might make you believe, this function is identical to `0x808359A4` so we mark the address for both here.
```suggestion
/// @addr{0x808357E4} @addr{0x808359A4}
```